components/suggestion/command_provider/app_command_parser/app_command_parser.ts

```diff
@@ -1183,7 +1183,8 @@ export class AppCommandParser {
             }));
         }
 
-        const items = callResponse?.data?.items;
+        let items = callResponse?.data?.items;
+        items = items?.filter((option) => option.value && !option.value.match(/^[ \t]+$/));
```

Comment:
Does it make sense to move the filtering into a helper function to keep the code DRY?